pkgs/development/python-modules/myfitnesspal/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, fetchPypi, buildPythonPackage
-, blessed, keyring, keyrings-alt, lxml, measurement, python-dateutil, requests, six
-, mock, nose }:
+, blessed, keyring, keyrings-alt, lxml, measurement, python-dateutil, requests, six, rich
+, pytest, mock, nose }:
```

Comment:
```suggestion
, blessed, keyring, keyrings-alt, lxml, measurement, python-dateutil, requests, six,
, rich, pytest, mock, nose }:
```

Reply:
I think this is ordered based on `propagatedBuildInputs` vs `checkInputs`, and `rich` is an `propagatedBuildInput`. I think it should be as I've done?

Reply:
I've seen it done both ways, sorted by usage and sorted alphabetically. Both are fine from what I've read.